src/main/webapp/resources/js/pages/projects/samples/linker/Linker.jsx

```diff
@@ -27,102 +27,85 @@ const CommandText = styled(Paragraph)`
  * @constructor
  */
 function Linker() {
+  const [visible, setVisible] = useState(false);
+  const [types, setTypes] = useState(["fasta"]);
```

Comment:
`fastq` not `fasta` for the default please.

Reply:
Changed in 46931f5fb80dc24ea9a492508e01aa606963192c